api/v1beta2/awscluster_types.go

```diff
@@ -226,6 +231,18 @@ type AWSLoadBalancerSpec struct {
 	PreserveClientIP bool `json:"preserveClientIP,omitempty"`
 }
 
+// AdditionalListenerSpec defines the desired state of an
+// additional listener on an AWS load balancer.
+type AdditionalListenerSpec struct {
+	// Port sets the port for the additional listener.
+	Port int64 `json:"port"`
+	// Protocol sets the protocol for the additional listener.
+	// Currently only TCP is supported.
+	// +kubebuilder:validation:Enum=TCP
+	// +kubeBuilder:default=TCP
+	Protocol ELBProtocol `json:"protocol"`
```

Comment:
A default required the field to be optional and use `omitempty`.

Do CAPI projects currently generate OpenAPI json? If so, you need a secondary default tag, I think for now you don't and this can be fixed later, it's only needed for generated apply configuration for SSA

Reply:
I don't think so